### ISSUE TITLE
Hey Hey...

### ISSUE THREAD

**Issue by VaniiillaICE** [eiksch/statev_v2_issues#2828]
## WICHTIGE INFORMATION!!:
```
Hier sollen nur Bugs gemeldet werden, Vorschläge bitte ins StateV-Forum!
```

**Character Name**

**Auftrittszeitpunkt (Datum / Uhrzeit)**

**Beobachtetes Verhalten**

**Erwartetes Verhalten**

**Schritte um den Fehler nachvollziehen zu können**

**Monitorauflösung (nur wenn Falsche Darstellung in der UI)**

**Optional: Video / Bilder des Fehlers**

**Comment by CaleighRhodes** [comment]
Bezüglich der Kette und der Ohrringe: #687

Alle Creolen kannst du nicht mit Ketten kombinieren, weil sie auf dem gleichen Slot liegen. Wenn du aber bspw. Diamantohrstecker oder die hängenden Ohrringe nimmst, geht das miteinander.

**Comment by PlayInc89** [comment]
Versuch bei dem Oberteil Kurzarm 2 sollte eig. gehen.

**Comment by VaniiillaICE** [comment]
Ne hab alle durch versucht auch Kurzarm2..bei der alten frisur ging es nur bei der gibts keine passende Anpassung, die Löcher bleiben.

**Comment by CaleighRhodes** [comment]
> Und bezüglich der Ohrringe nochmal @CaleighRhodes Kette wie auch die Ohrringe gingen beide in den richtigen Slot nur wurde dann immer jeweils eins von beiden unsichtbar. Also es wird schon erkannt ob Kette oder Ohrringe. Aber wenn ihr da schon Bescheid wisst hab ich nix gesagt :3

Ja, ist richtig. Lies dir mal das durch, was ich verlinkt habe. Man kann bei Creolen und Muschelohrringen nicht zeitgleich eine Kette tragen. Du kannst sie zwar anziehen, sie wird aber nicht angezeigt. Also jeweils das Erste, was du angelegt hast.
